### PR TITLE
Deposit ERC20 in crab V2

### PR DESCRIPTION
**Pull request by KMKoushik**
# Task:

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Document update

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/hardhat/contracts/strategy/CrabHelper.sol [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+import {ICrabStrategyV2} from "../interfaces/ICrabStrategyV2.sol";
+import {IWETH9} from "../interfaces/IWETH9.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import {Address} from "@openzeppelin/contracts/utils/Address.sol";
+
+import {StrategySwap} from "./helper/StrategySwap.sol";
+
+/**
```
Comment (alpinechicken) [inline packages/hardhat/contracts/strategy/CrabHelper.sol:12]: tests are complaining about a license

**Review by alpinechicken — APPROVED** [review]
lgtm

**Review by aparnakr — APPROVED** [review]
Looks good to merge

File: packages/hardhat/test/integration-tests/crabv2/crab-helper.ts [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+  this.beforeAll("Deploy Crab Helper", async () => {
+    const CrabHelperContract = await ethers.getContractFactory("CrabHelper");
+    crabHelper = (await CrabHelperContract.deploy(crabStrategy.address, swapRouter.address)) as CrabHelper;
+  })
+
+  describe("Deposit USDC into strategy", async () => {
+    const usdcAmount = startingEthPrice1e18
+    const ethToDeposit = ethers.utils.parseUnits('1.5')
+
+    beforeEach(async () => {
```
[the remark this answers is not in the document]
Comment (KMKoushik) [reply packages/hardhat/test/integration-tests/crabv2/crab-helper.ts:137]: Want it before each tests

**Pull request merged**